sr_multi_moveit/sr_multi_moveit_config/launch/demo.launch

```diff
@@ -0,0 +1,69 @@
+<launch>
+  <arg name="planning" default="ompl"/>
+  <!-- By default, we are not in debug mode -->
+  <arg name="debug" default="false" />
+  <!-- By default, we do not start a database (it can be large) -->
+  <arg name="db" default="false" />
+
```

Comment:
maybe instead of commenting these out, have an arg and some if statements? maybe?